ui/src/components/table/table-column-selection.js

```diff
@@ -2,7 +2,13 @@ import { isNumber } from '../../utils/is.js'
 
 export default {
   props: {
-    visibleColumns: Array
+    visibleColumns: Array,
+    labelFormat: {
+      type: Function,
+      default (name) {
+        return name.toUpperCase()
```

Comment:
I think the default here would be to just `return name`

Reply:
I see, it's because of this was removed and was the default:
```js
label: name.toUpperCase(),
```

Reply:
Yeah, it used to be the default. So I used that again,